frontend/src/lib/components/BillingToolbar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { WarningOutlined, ToolFilled } from '@ant-design/icons'
+import { Button } from 'antd'
+
+export function BillingToolbar({
+    billingUrl = null,
+    message,
+}: {
+    billingUrl: string | null
+    message: string
+}): JSX.Element {
+    return (
+        <>
+            {billingUrl && (
+                <div className="card">
+                    <div className="card-body" style={{ display: 'flex' }}>
```

Comment:
I would just add a hook here to `const { user } = useValues(userLogic)`, so there's no need for passing props